addon/components/base/bs-form.js

```diff
@@ -44,6 +44,7 @@ import layout from 'ember-bootstrap/templates/components/bs-form';
     {{form.element controlType="email" label="Email" placeholder="Email" property="email"}}
     {{form.element controlType="password" label="Password" placeholder="Password" property="password"}}
     {{form.element controlType="checkbox" label="Remember me" property="rememberMe"}}
+    {{bs-button defaultText="Validate" type="info" onClick=(action form.validate)}}
```

Comment:
I think it would be nice to have this new feature mentioned a bit more explicitly, maybe the yielded `form.submit` action under the `Submitting the form` chapter, and `validate` under `Form validation`?